CONTRIBUTING.md

```diff
@@ -8,28 +8,86 @@ As a condition of providing a Contribution, you agree that:
 - You irrevocably grant anyone the right to use your work under the following license: Creative Commons CC0 Waiver (release all rights, like public domain: [legal code](https://creativecommons.org/publicdomain/zero/1.0/))
 - You warrant and represent that the Contribution is your original creation, that you have the authority to grant this license to anyone, and that this license does not require the permission of any third party. Otherwise, you provide your Contribution "as is" without warranties.
 
+# How to translate
 
-# Ways to translate
-## Which way
 There are multiple ways to add or edit your translation of the game:
-1. The easy way
-  * Edit the repository directly through GitHub's web editor. Navigate to a file, click on the edit-button on the top far right, make your changes, add a description click on the 'Propose changes'.
-  * Pro: fast and easy.
-  * Con: not suited for bigger changes, as the web-editor has much less tools than a full fledged text editor.
-2. The manual way
-  * [Download](https://github.com/ReactiveDrop/reactivedrop_translations/archive/refs/heads/master.zip) the repository, edit it on your computer and then submit it through the [Issues tab](https://github.com/ReactiveDrop/reactivedrop_translations/issues) by providing an archive.
-  * Pro: you do not have to understand how to work with [Git](https://en.wikipedia.org/wiki/Git).
-  * Con: the admins have to do the git-work for you, and as a result your changes will be logged as theirs.
-3. The professional way
-  * [Fork the repository](https://github.com/ReactiveDrop/reactivedrop_translations/fork), edit & commit it and then push your changes back to the origin.
-  * Pro: full control over big sets of changes, use functions like search & replace in files, regex, machine translation.
-  * Con: steeper learning curve using the git system.
-  * Use an editor to make your changes and integrate them through github. Read more about it [on our wiki](https://github.com/ReactiveDrop/reactivedrop_translations/wiki).
-
-## Pre-requisites
+
+* via the GitHub web editor [\[jump\]]()
+* by downloading and editing an archive of the repository [\[jump\]]()
+* by forking the Git repository, editing your fork, and then submitting a pull request [\[jump\]]()
+
+## Using the GitHub web editor
+
+0. Ensure you are [logged in](https://github.com/login) to GitHub.
+
+1. Open the [GitHub web editor](https://github.dev/ReactiveDrop/reactivedrop_translations) by clicking on this link.
+
+2. When prompted, install the suggested extension.
+
+   ![Recommended Extensions tooltip, suggesting the installation of "Valve KeyValue File Support".](https://i.imgur.com/2YjF5ue.png)
```

Comment:
would probably be better to upload the screenshots to a folder in this repo or to put them in developer.reactivedrop.com or something (not sure if imgur images are going to stay around; they do auto-delete images occasionally)

Reply:
do they?  
I've never seen imgur deleting any image, and I can't find anything on the web about it

I agree that uploading them somewhere else would be a wise idea, but I'd avoid putting them in the repository, as that would make it a quite bit larger in file size...

Reply:
I'd recommend developer.reactivedrop.com, then (it's a github pages site, so the repo just gets copied to a CDN).

I'm pretty sure I'm the only one with any reason to have developer.reactivedrop.com's repo downloaded right now, so no worries about people having to download the images repeatedly.

Reply:
This should let us avoid going through GitHub Pages by using GitHub User Content as a CDN directly!